README.md

```diff
@@ -1,145 +1,105 @@
-# Planner
-
-A tool to help manage [codebar.io](http://codebar.io) members and events.
+# codebar website & workshop planner
```

Comment:
Can we change `workshop` to `event` please.

Reply:
@richardwestenra are you happy for me to make this change so we can merge this PR in?